### PR TITLE
Add a convenience script for running individual tests

### PR DESCRIPTION
**Pull request by wilzbach**
After typing `test_results` and adding `.out` over an over again, I was so annoyed that I finally automated this and though this is worth sharing as others might miss this too.

Usage:

```sh
./run_single runnable/template2962.d fail_compilation/fail282.d
```

Reasons:
- safe time
- auto-completion in CLI works

If we plan to move the documentation to a README.md as proposed [here](https://github.com/dlang/dmd/pull/8038#issuecomment-373524698), this could also be documented though I think it's rather self-explanatory.

This might be a small addition, but it saves a bit of time for me every time and that quickly adds up.

### CODE REVIEW

File: test/run_single [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Allows to quickly run a single tests
```
Comment (RazvanN7) [inline test/run_single]: "Allows to run single tests". The use of quickly is redundant

**Review by RazvanN7 — COMMENTED** [review]
Loooks good, but I would rename it `run_custom_tests`

**Comment by wilzbach** [comment]
> I would rename it run_custom_tests

Ok. (I don't really care about the name of this script)

**Comment by jacob-carlborg** [comment]
> Loooks good, but I would rename it run_custom_tests

`custom` sounds to me like my own test that I personally want to run that no one else runs.

**Comment by wilzbach** [comment]
> custom sounds to me like my own test that I personally want to run that no one else runs.

(I'm happy about better name suggestion - as mentioned I don't really care about the name)

**Comment by RazvanN7** [comment]
>  custom sounds to me like my own test that I personally want to run that no one else runs.

`run_selected_tests` , `run_hand_picked_tests` , `run_chosen_tests` ?

**Comment by JinShil** [comment]
Yeah, `run_custom_tests` isn't right, `run_selected_tests` isn't bad, but I would prefer `run_individual_tests`.

**Comment by wilzbach** [comment]
> I would prefer run_individual_tests.

OK. Renamed. Anything else blocking this?

**Review by JinShil — APPROVED** [review]

**Pull request merged**